### DISCUSSION TITLE
[Bug] 安装新版本之后，路由切换出现了问题。

### DISCUSSION THREAD

**Discussion by BH-M87** [umijs/umi#10029] [Q&A]
如果你也遇到了，可以留个言。

**Comment by fz6m** [comment]
是 qiankun 子路由相关的吗，给个复现看看。

**Reply by BH-M87** [reply]
项目比较大，估计不好复现。

问题出在history.push这个方法上。切换路由的时候调用的history.push。

**Reply by 426-330** [reply]
更新Umi4后，使用路由绑定引入子应用时，在页面中通过事件切换路由时，子应用的页面没有更新，刷新页面正常
